jobs/webcompat-kb/webcompat_kb/main.py

```diff
@@ -1105,17 +1172,31 @@ def run(self) -> None:
             last_change_time_max,
         )
 
+        self.update_metric_history()
 
-@click.command()
-@click.option("--bq_project_id", help="BigQuery project id", required=True)
-@click.option("--bq_dataset_id", help="BigQuery dataset id", required=True)
-@click.option(
-    "--bugzilla_api_key", help="Bugzilla API key", required=False, default=None
-)
-def main(bq_project_id: str, bq_dataset_id: str, bugzilla_api_key: str) -> None:
-    logging.getLogger().setLevel(logging.INFO)
 
-    bz_bq = BugzillaToBigQuery(bq_project_id, bq_dataset_id, bugzilla_api_key)
+def get_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--log-level",
+        choices=["debug", "info", "warn", "error"],
+        default="info",
+        help="Log level",
+    )
+    parser.add_argument("--bq-project-id", required=True, help="BigQuery project id")
```

Comment:
should probably keep this with underscore as it's being passed in airflow https://github.com/mozilla/telemetry-airflow/blob/main/dags/webcompat_kb.py#L53